feedback-module/src/components/Module.js

```diff
@@ -161,37 +149,35 @@ function Module({ pagetitle, endpoint, dir }) {
       <GridContainer
         desktop={{ col: 2 }}
         mobile={{ col: "fill" }}
-        className={MODULE_CONTAINER_STYLE}
+        className="feedback-module-container"
         dir={dir}
       >
         {moduleOnScreen(moduleVisibleRef)}
         <Header innerRef={headerRef} />
         {screen.titleInverse && (
-          <Grid className={`bg-primary ${SCREEN_CONTAINER_STYLE}`}>
+          <Grid className={"bg-primary feedback-screen-container"}>
             <p
-              className={`${H1_WHITE_STYLE} ${dir === "rtl" && "text-right"}`}
+              className="font-sans-md2 feedback-h1 feedback-h1--light"
```

Comment:
Do we need both "feedback-h1 and feedback-h1--light"? Isn't the purpose of using extends for --light so that it automatically inherits the styles from feedback-h1?

Reply:
I'd considered just using `feedback-h1--light`, but according to the official BEM docs, we should keep in the original class and add the new one separately (see http://getbem.com/naming/ Modifier section).